src/components/ComponentExample.tsx

```diff
@@ -10,11 +10,12 @@ const ComponentExample = ({ style, children }: Props): JSX.Element => (
       flexWrap: 'wrap',
       padding: suomifiTheme.spacing.m,
       margin: `${suomifiTheme.spacing.m} 0`,
+      border: `1px solid ${suomifiTheme.colors.depthBase}`,
```

Comment:
So this sets the default border if none is defined in style prop. Could this be handled by destructuring border from props and checking if it's provided? Then it would be possible to either use the default or provided - now the default is overriden by what is provided with style and requires default definition in button as well?

Reply:
=> providing no border or undefined / null border via props should cause the default to take over and remove the need for double definition?

Reply:
Cleaned it up now. Mobile button example didn't use the ComponentExample component so had to add the style manually to that one, but the rest are cleaner now and adding an empty or undefined border style doesn't override the default styling.